src/Controls/src/Core/Xaml/TypeConversionExtensions.cs

```diff
@@ -217,16 +216,11 @@ static Type GetConverterType(this TypeConverterAttribute attribute)
 				}
 			}
 
-			//if the value is not assignable and there's an implicit conversion, convert
 			if (value != null && !toType.IsAssignableFrom(value.GetType()))
 			{
-				var opImplicit = value.GetType().GetImplicitConversionOperator(fromType: value.GetType(), toType: toType)
-								?? toType.GetImplicitConversionOperator(fromType: value.GetType(), toType: toType);
-
-				if (opImplicit != null)
+				if (TypeConversionHelper.TryConvert(value, toType, out var convertedValue))
```

Comment:
Technically this does change the existing behavior - if there was a TypeConverter, it would not have been used, but now it will be, right?

Reply:
Yes, if the existing type converter can convert between the two types, it would be used. But that's the same case as in `BindableProperty.TryConvert`. If we wanted to make sure that behavior doesn't change, we would have to never use type converters when the `MauiImplicitCastOperatorsUsageViaReflectionSupport ` feature switch is not set.

This behavior would definitely make it even more in line with "don't break any existing app", but maybe it would make adoption of type converters instead of implicit casts slower? This is a good question and I'm not sure which option I prefer.